testdata/time.go

```diff
@@ -11,3 +11,27 @@ import (
 var rpcTimeoutMsec = flag.Duration("rpc_timeout", 100*time.Millisecond, "some flag") // MATCH /Msec.*\*time.Duration/
 
 var timeoutSecs = 5 * time.Second // MATCH /Secs.*time.Duration/
+
+var delay = flag.Duration("delay", 5, "wait impatiently") // MATCH /time\.Duration without unit: 5/
+
+func f(...time.Duration) {}
+
+var (
+	_ = time.Duration(15)
```

Comment:
I saw a lot of `time.Duration(30) * time.Second` when I was flagging it, which is maybe not ideal, but seemed intentional and not the end of the world. It also would need special handling to catch. I'll add that handling if you would like--let me know.